conda/core/path_actions.py

```diff
@@ -82,20 +83,27 @@
 )
 
 
-class _Action(metaclass=ABCMeta):
+class Action:
```

Comment:
Wouldn't it be easier to make this the base Abstract class? If we did that, we wouldn't have to duplicate the abstract method definitions in `FinalTransactionAction`.